packages/grafbase-sdk/src/typedefs/auth.ts

```diff
@@ -59,6 +60,13 @@ export class AuthDefinition {
     return new CacheDefinition(this, new FieldLevelCache(params))
   }
 
+  /**
+   * Sets the name of the field in the database, if different than the name of the field.
+   */
+  public map(name: string): MapDefinition {
```

Comment:
A random thought: `map` might be an odd name for this in a TS context - quite different from the usual `.map` function.